compiler/x/codegen/FPTreeEvaluator.cpp

```diff
@@ -562,27 +561,10 @@ TR::Register *OMR::X86::TreeEvaluator::fpBinaryArithmeticEvaluator(TR::Node
                                                               bool              isFloat,
                                                               TR::CodeGenerator *cg)
    {
-   static auto Disable3OpForFP = (bool)feGetEnv("TR_Disable3OpForFP");
-   if (!Disable3OpForFP && cg->useSSEForSinglePrecision() && cg->useSSEForDoublePrecision())
+   if (cg->useSSEForSinglePrecision() && cg->useSSEForDoublePrecision())
       {
       return TR::TreeEvaluator::FloatingPointAndVectorBinaryArithmeticEvaluator(node, cg);
       }
-   // Attempt to use SSE/SSE2 instructions if the CPU supports them, and
```

Comment:
Could you provide a short summary of why this code was never going to be used ?

Reply:
`FloatingPointAndVectorBinaryArithmeticEvaluator` always generates same or better code than `XMMBinaryArithmeticAnalyser`. `XMMBinaryArithmeticAnalyser` never leverage 3-operand form instruction or the RegMem form instructions.